stream_alert_cli/terraform_generate.py

```diff
@@ -352,6 +352,32 @@ def generate_cloudtrail(cluster_name, cluster_dict, config):
     """
     modules = config['clusters'][cluster_name]['modules']
     cloudtrail_enabled = bool(modules['cloudtrail']['enabled'])
+    existing_trail_default = False
+    existing_trail = modules['cloudtrail'].get('existing_trail', existing_trail_default)
+    is_global_trail_default = True
```

Comment:
Maybe I'm just not seeing it elsewhere, but it looks like `existing_trail_default` and `is_global_trail_default` are just booleans and only used to hold default values for the `.get()` calls. Can we just place the boolean value directly in the get for brevity?